src/main/java/app/bpartners/geojobs/repository/model/community/CommunityAuthorization.java

```diff
@@ -0,0 +1,37 @@
+package app.bpartners.geojobs.repository.model.community;
+
+import static jakarta.persistence.CascadeType.ALL;
+
+import jakarta.persistence.*;
+import java.io.Serializable;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name = "community_authorization")
+public class CommunityAuthorization implements Serializable {
+  @Id private String id;
+
+  @Column private String name;
+
+  @Column(name = "api_key")
```

Comment:
Why do you specify explicitly each column name but doesn't let hibernate map it automatically ?